build-go/pkg/build.go

```diff
@@ -82,9 +89,20 @@ func (b *GoBuild) Run() error {
 	if err != nil {
 		return err
 	}
-	flags = append(flags, []string{"-o", filename}...)
 
-	fmt.Printf("::set-output name=go-binary-name::%s\n", filename)
+	flags = append(flags, []string{"-o", binary}...)
+
+	// A dry run prints the information that is trusted, before
```

Comment:
trusted? aren't these supplied by the user in the config? (the binary name)

Reply:
yes they are, but the print can be trusted because we have not run the compiler using the user-supplied flags yet, so we can trust that the command that will be run is the one that the dry run reports.
Does it make sense? Shall I update the comment?

Reply:
AH I gotcha! I get why you say "trusted, before the compiler is invoked" now. Nw!